clang/lib/CodeGen/CGExprComplex.cpp

```diff
@@ -1356,6 +1361,7 @@ LValue ComplexExprEmitter::EmitBinAssignLValue(const BinaryOperator *E,
 }
 
 ComplexPairTy ComplexExprEmitter::VisitBinAssign(const BinaryOperator *E) {
+  ApplyAtomGroup Grp(CGF.getDebugInfo());
   ComplexPairTy Val;
   LValue LV = EmitBinAssignLValue(E, Val);
```

Comment:
Similarly, if we followed the pattern of the above hunk, we'd put ApplyAtomGroup into `EmitBinAssignLValue` instead. This one covers the load-of-lvalue at the end, wheras for compound assignments the equivalent wouldn't be covered.

Reply:
I'm a bit confused, the previous patch puts the ApplyAtomGroup in `ScalarExprEmitter::VisitBinAssign`, so this looks symmetrical to me - am I missing something?